src/app/components/pokemon-card/pokemon-card.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="card" (click)="onSelect(pokemon)">
+  <img
+    class="card__img"
+    src="{{pokemon.imageUrl}}"
```

Comment:
use inputs instead of interpolation

Reply:
Do you refer to the src attribute? I think it is more inconvenient

Reply:
Done